sccp/figures/common.py

```diff
@@ -140,8 +140,39 @@ def plotFactors(factors, data: Pf2X, axs, reorder=tuple(), trim=tuple(), saveGen
             if i == 2 and len(yt) > 50:
                 df = pd.DataFrame(data=X, index=yt, columns=[f"Cmp. {i}" for i in np.arange(1, rank + 1)])
                 df.to_csv(join(path_here, "data/TopBotGenes_Cmp"+str(rank)+".csv"))
-   
                 
+def plotCondFactorsReorder(factors, data: Pf2X, ax):
```

Comment:
@aarmey Is this what you were referring to from our discussion yesterday from identifying which drugs have the biggest effects? This doesn't provide much more insight from the original factors

Reply:
I was wondering if, by taking the max you observe for each drug across components, this could provide an easy way to visualize which drugs do or do not have a specific effect.